src/Components/WebAssembly/Build/src/targets/ServiceWorkerAssetsManifest.targets

```diff
@@ -39,7 +39,7 @@
   <Target Name="_ComputeServiceWorkerAssetsManifestFileHashes">
 
     <ItemGroup>
-      <ServiceWorkerAssetsManifestItem Include="%(StaticWebAsset.Identity)" Condition="'%(RelativePath)' != '$(ServiceWorkerAssetsManifest)'">
+      <ServiceWorkerAssetsManifestItem Include="%(StaticWebAsset.Identity)" Condition="'%(RelativePath)' != '$(ServiceWorkerAssetsManifest)' and %(Extension) != '.gz'">
```

Comment:
We filter on the extension for the time being, although we could have constructed this itemgroup with `%(BasePath)%(RelativePath)` and `KeepDuplicates="false"` which represents the endpoint at which the file will be available.

I plan to generate such an itemgroup in preview3 and we can rely on that here.